seaclouds.yaml

```diff
@@ -55,9 +55,35 @@ services:
             DB_PASSWORD: "_atossla_"
       war: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=sla-service&v=LATEST&e=war
 
-
     - serviceType: brooklyn.entity.database.mysql.MySqlNode
       id: sla-db
       name: SLA Db
       brooklyn.config:
         creationScriptUrl: https://raw.githubusercontent.com/SeaCloudsEU/sla-core/e1d3bd4dec27236cfdefa1eae81d38db3dcd11da/sla-repository/src/main/resources/sql/01database.sql
+
+    - serviceType: brooklyn.entity.basic.VanillaSoftwareProcess
+      name: SLA Dashboard
+      id: sla-dashboard
+      launch.command: |
+        # Listen socket
+        export SOCKET=0.0.0.0:8000
+        # install necessary packages
+        sudo apt-get -y install python-pip
+        sudo apt-get -y install git
+        sudo pip install virtualenv
+        # setup application environment
+        export VIRTUALENVS_DIR=$HOME/virtualenvs
+        export SLA_DASHBOARD=$PWD
+        rm -Rf $VIRTUALENVS_DIR/sla-dashboard
+        # install application
+        git clone https://github.com/SeaCloudsEU/sla-dashboard .
+        virtualenv $VIRTUALENVS_DIR/sla-dashboard
+        . $VIRTUALENVS_DIR/sla-dashboard/bin/activate
+        pip install -r requirements.txt
+        ./manage.py syncdb --noinput
+        # run application
+        ./manage.py runserver $SOCKET &
+        echo $! > $PID_FILE
+      env:
+        SLA_MANAGER_URL: $brooklyn:formatString("http://%s:%s", component("sla-core").attributeWhenReady("host.address"), component("sla-core").attributeWhenReady("http.port"))
```

Comment:
What url? SLA_MANAGER_URL? I do use it. But sth strange happens, because brooklyn tries to reassign it each minute.